sdv/data_processing/data_processor.py

```diff
@@ -103,6 +104,104 @@ def __init__(self, metadata, enforce_rounding=True, enforce_min_max_values=True,
         if self._primary_key:
             self._keys.append(self._primary_key)
 
+    def _get_column_in_multi_column_transformer(self):
+        """Get the columns that are part of a multi column transformer.
+
+        Returns:
+            list:
+                A list of columns that are part of a multi column transformer.
+        """
+        return [
+            col for col_tuple in self.columns_to_multi_col_transformer for col in col_tuple
+        ]
+
+    def _import_address_transformers(self):
+        """Try to import the address transformers."""
+        try:
+            from rdt.transformers import RandomLocationGenerator, RegionalAnonymizer
+            self.RandomLocationGenerator = RandomLocationGenerator
+            self.RegionalAnonymizer = RegionalAnonymizer
+        except ImportError:
+            raise ImportError(
+                'You must have SDV Enterprise with the address add-on to use the address features'
+            )
+
+    def _get_columns_in_address_transformer(self):
+        """Get the columns that are part of the address transformer.
+
+        Returns:
+            list:
+                A list of columns that are part of the address transformers.
+        """
+        return [
+            col_tuple for col_tuple, transformer in self.columns_to_multi_col_transformer.items()
+            if isinstance(transformer, (self.RandomLocationGenerator, self.RegionalAnonymizer))
+        ]
+
+    def _get_address_transformer_parameters(self, column_names):
+        """Get the parameters for the address transformer.
+
+        Args:
+            column_names (tuple[str]):
+                The column names to get the parameters for.
+
+        Returns:
+            dict:
+                A dictionary mapping column names to sdtypes.
+        """
+        columns_to_sdtypes = {}
+
+        for column in column_names:
+            sdtype = self.metadata.columns[column]['sdtype']
+            columns_to_sdtypes[column] = sdtype
+
+        return columns_to_sdtypes
+
+    def _get_address_transformer(self, anonymization_level):
+        """Get the address transformer.
+
+        Args:
+            anonymization_level (str):
+                The anonymization level for the address transformer.
+        """
+        transformer = None
+        if anonymization_level == 'full':
+            transformer = self.RandomLocationGenerator()
+        elif anonymization_level == 'street_address':
+            transformer = self.RegionalAnonymizer()
```

Comment:
this should be `rdt.transformers.RandomLocationGenerator`
OR in the worst case use lazy import if that is something that you want to achieve here, but I wouldn't vouch for that.